microsite/static/css/custom.css

```diff
@@ -48,6 +48,14 @@ h6 {
   color: $textColor;
 }
 
+h2:hover .hash-link {
+  opacity: 1
```

Comment:
yep there's a missing semicolon here

Reply:
I don't know why that didn't get picked when I ran the command locally, my bad